plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list-item/payment-gateway-list-item.tsx

```diff
@@ -0,0 +1,122 @@
+import { WooPaymentMethodsLogos } from '@woocommerce/onboarding';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { decodeEntities } from '@wordpress/html-entities';
+
+/**
+ * Internal dependencies
+ */
+import { StatusBadge } from '~/settings-payments/components/status-badge';
+import { PaymentGatewayButton } from '~/settings-payments/components/payment-gateway-button';
+import { WooPaymentsGatewayData } from '~/settings-payments/types';
+import { useState } from 'react';
+
+type PaymentGatewayItemProps = {
+	gateway: PaymentGateway;
+	wooPaymentsGatewayData?: WooPaymentsGatewayData;
+	setupLivePayments: () => void;
+};
+
+export const PaymentGatewayListItem = ( {
+	gateway,
+	wooPaymentsGatewayData,
+	setupLivePayments,
+}: PaymentGatewayItemProps ) => {
+	const [ isEnabled, setIsEnabled ] = useState( gateway.enabled );
+
+	const determineGatewayStatus = () => {
+		if ( ! isEnabled && gateway?.needs_setup ) {
+			return 'needs_setup';
+		}
+		if ( isEnabled ) {
+			if ( gateway.id === 'woocommerce_payments' ) {
+				if ( wooPaymentsGatewayData?.isInTestMode ) {
+					return 'test_mode';
+				}
+			}
+			return 'active';
+		}
+
+		if ( gateway.id === 'woocommerce_payments' ) {
+			return 'recommended';
+		}
+
+		return 'inactive';
+	};
+
+	return {
+		key: gateway.id,
+		title: (
+			<>
+				{ gateway.method_title }
+				<StatusBadge status={ determineGatewayStatus() } />
+			</>
+		),
+		content: (
+			<>
+				{ decodeEntities( gateway.method_description ) }
+				{ gateway.id === 'woocommerce_payments' && (
+					<WooPaymentMethodsLogos
+						maxElements={ 10 }
+						isWooPayEligible={ true }
+					/>
+				) }
+			</>
+		),
+		after: (
+			<div className="woocommerce-list__item-after__actions">
+				<>
+					<PaymentGatewayButton
+						id={ gateway.id }
+						enabled={ isEnabled }
+						needs_setup={ gateway.needs_setup }
+						settings_url={ gateway.settings_url }
+						setIsEnabled={ setIsEnabled }
+					/>
+					{ gateway.id === 'woocommerce_payments' &&
+						wooPaymentsGatewayData?.isInTestMode && (
+							<Button
+								variant="primary"
+								onClick={ setupLivePayments }
+								isBusy={ false }
+								disabled={ false }
+							>
+								{ __( 'Set up live payments', 'woocommerce' ) }
+							</Button>
+						) }
+					<EllipsisMenu
+						label={ __( 'Task List Options', 'woocommerce' ) }
+						renderContent={ () => (
+							<div>
+								<Button>
+									{ __( 'Learn more', 'woocommerce' ) }
+								</Button>
+								<Button>
+									{ __(
+										'See Terms of Service',
+										'woocommerce'
+									) }
+								</Button>
+							</div>
+						) }
+					/>
+				</>
+			</div>
+		),
+		// TODO add drag-and-drop icon before image (future PR)
+		before: (
+			<img
+				src={
+					// TODO: Need a way to make images available here.
+					'https://woocommerce.com/wp-content/plugins/wccom-plugins/payment-gateway-suggestions/images/wcpay.svg'
```

Comment:
Not releated to this PR, but just a note: I think we can use the existing WCPay image [here](
https://github.com/woocommerce/woocommerce/blob/6f7c494d38002789631b5a44dd0868b320c1182c/plugins/woocommerce/assets/images/onboarding/wcpay.svg).

```js
import { WC_ASSET_URL } from '~/utils/admin-settings';
....
	src={ `${ WC_ASSET_URL }images//onboarding/wcpay.svg` }
```

Reply:
good catch! used in https://github.com/woocommerce/woocommerce/pull/52576/commits/802e820e943fe8433de406412adf21835318e7b0